backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/imagegallery/BpkImageGallery.kt

```diff
@@ -51,9 +53,113 @@ fun BpkImageGallery(
     }
 }
 
+@Composable
+fun BpkImageGalleryChipGrid(
```

Comment:
created two variants for image vs grid - api is basically the same except for the category type/callback type